### PR TITLE
netdata: remove judy from buildInputs as it is vendored by upstream

### PR DESCRIPTION
**Pull request by NickCao**
###### Description of changes
See https://github.com/netdata/netdata/pull/13362, unfortunately their decision to do so also breaks cross compilation.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by viraptor** [comment]
Result of `nixpkgs-review pr 186554` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package marked as broken and skipped:</summary>
  <ul>
    <li>netdata</li>
  </ul>
</details>

**Review by rapenne-s — APPROVED** [review]
LGTM

File: pkgs/tools/system/netdata/default.nix [hunk 1]
```diff
@@ -37,7 +37,7 @@ in stdenv.mkDerivation rec {
     ++ optionals stdenv.isDarwin [ CoreFoundation IOKit libossp_uuid ]
     ++ optionals (!stdenv.isDarwin) [ libcap.dev libuuid.dev ]
     ++ optionals withCups [ cups ]
-    ++ optionals withDBengine [ judy lz4.dev ]
+    ++ optionals withDBengine [ lz4 ]
```
Comment (winterqt) [inline pkgs/tools/system/netdata/default.nix:40]: Is the `lz4.dev` -> `lz4` change related to this? (If not, I'm confused as to why it's suddenly not needed.)

Comment (NickCao) [reply pkgs/tools/system/netdata/default.nix:40]: I believe it shouldn't be needed in the first place.

Comment (NickCao) [reply pkgs/tools/system/netdata/default.nix:40]: According to the build log, lz4 is still correctly found.
```
netdata> checking for LZ4_initStream in -llz4... yes
netdata> checking for LZ4_compress_default in -llz4... yes
```

Comment (winterqt) [reply pkgs/tools/system/netdata/default.nix:40]: Wonder if it's required for `libcap` and `libuuid`, you'd think not. Hm.

Comment (NickCao) [reply pkgs/tools/system/netdata/default.nix:40]: stdenv takes care of selecting the correct output most of the times.

Comment (winterqt) [reply pkgs/tools/system/netdata/default.nix:40]: Yes, which is why I think if we're cleaning it up for one, might as well do it for all of them in this PR. Maybe a separate commit, though -- not sure.

**Comment by winterqt** [comment]
Also: maybe we should add `broken = stdenv.buildPlatform != stdenv.hostPlatform`, to bail out if cross-compiling?

**Pull request merged**